src/pages/Aggregator/Aggregator.component.tsx

```diff
@@ -40,12 +44,21 @@ const PageViewTitle: React.FC = ({ children }) => (
   </Box>
 );
 
+const slippageSliderValues = [0, 0.5, 1, 1.5, 2, 2.5];
```

Comment:
Gotta update the spec

Reply:
@dharkmattr why should the spec be updated?

Reply:
The meaning of these numbers is the point change in fee. So if, for example, I deposit $1000 and get a reward of $1, the maximum tolerance of 1% means that a transaction in which the reward is lesser than $0.99 will fail. That's a slim tolerance and would mean, most likely, that most transactions will fail. 

Thinking from the user's perspective, why would I not be allowed to make a deposit and just be happy with whatever I get?

Reply:
@dharkmattr the values were copied from the slippage tolerance settings used for the Sovryn AMM. They were a first approximation of what the numbers should be, prior to knowing what the actual reward/penalty parameters will be. After considering the discussions we've had about this since the PRD was first written last year, and considering your suggestion above, I propose this adjustment to the spec for this UI feature:

- The slippage tolerance setting should define the maximum difference from the current reward/penalty applied by the smart contract e.g. if the user is depositing 1000 USDT and would receive a reward of 25 XUSD then a slippage tolerance setting of 5% would mean that the minimum reward the user will accept is 23.75 XUSD, otherwise the tx will revert.
  - This setting should only appear for transactions with Rootstock as the starting chain.
- The slippage tolerance section should include an amount entry field like the token amount entry field above it, with default selector buttons below with the values `1%, 1.5%, 2%, 2.5%, MAX`.
  - The user can either manually enter a value into the entry field, or click one of the selector buttons to enter a default value. If no value is entered, then the `MAX` value should be passed with the transaction.
  - `MAX` should result in a reward or penalty % equal to the `Max reward` or `Max penalty` based on the parameter in the contract, as applicable for the given chain/token selected.
- The user should be shown the `Penalty` and `Maximum penalty`, or `Reward` and `Minimum reward`, as applicable based on their selected slippage tolerance setting. This value should include the % of the reward or penalty which is equal to `Reward or penalty amount / Conversion amount` e.g. based on the example above, the user would see a `Estimated reward` of `25 XUSD (2.5%)` and `Minimum reward` of `23.75 XUSD (2.375%)`

WDYT?

Mockup:
![BF Slippage](https://github.com/BabelFishProtocol/babelfish-dapp/assets/9424721/b45b9f96-02ab-4502-91eb-30a4ddd1ec47)

Reply:
We decided to remove the slippage tolerance setting for now and use a sane default that is applied in the background. Spec on Jira has been updated to reflect this. We may re-implement the slippage tolerance setting as specified above in a future update.